service/dap/server.go

```diff
@@ -2031,9 +2053,11 @@ func (s *Server) onEvaluateRequest(request *dap.EvaluateRequest) {
 				}
 			}
 		}
-		// TODO(polina): as far as I can tell, evaluateName is ignored by vscode for expression variables.
-		// Should it be skipped altogether for all levels?
-		exprVal, exprRef := s.convertVariable(exprVar, fmt.Sprintf("(%s)", request.Arguments.Expression))
+		var opts convertVariableFlags
+		if ctxt == "variables" || ctxt == "hover" || ctxt == "clipboard" {
```

Comment:
Sadly "Copy Value" from WATCH appears to result in "watch" context request.

Reply:
Are you still getting this when running a dap server with supportsClipboardContext?